fortigate_exporter.go

```diff
@@ -18,54 +18,30 @@
 package main
 
 import (
-	"context"
-	"crypto/tls"
-	"crypto/x509"
-	"flag"
-	"fmt"
+	"github.com/bluecmd/fortigate_exporter/internal/config"
```

Comment:
Is there a reason to combine the imports? I would probably say it is more established practice to split core imports and 3rd party ones.

Reply:
adapted the pipeline and Makefile to handle this